e2e/cluster-debug-info.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
```

Comment:
At what point can this script just become a call to sos report?

Reply:
As soon as the sos version we need is publicly available this script may be retired.

Reply:
I don't remember the details, but I think we still don't have everything in SOS? Does it have "feature parity" with the cluster-debug-info.sh?

Reply:
I take ^ back. Just looked at sos from `master` and it's pretty dope.
@dhellmann @pacevedom what do you think about leaving couple commands in the job log to have quickly glanceable info in case of error? Like `oc get {pods,nodes}` and maybe `oc get events -A --sort-by=.metadata.creationTimestamp | head -n 20`. Given our problems with topolvm image pulling I'd get quickly annoyed to go through downloading and extracting sos report only to find that issue there :P